tests/application/controllers/signup.spec.ts

```diff
@@ -0,0 +1,505 @@
+import { InvalidParamError, MissingParamError, ServerError } from '../../../src/application/errors'
+import { SignUpController } from '../../../src/application/controllers/signup'
+import { type PhoneValidator, type EmailValidator, type NameValidator } from '../../../src/application/validation/protocols'
```

Comment:
Remove the "../../../src/" from the imports, because the `src` folder is already set as the root of the project in typescript